services/ekss.yaml

```diff
@@ -0,0 +1,19 @@
+name: ekss
+image: encryption-key-store-service
+description: Encryption Key Store Service
+summary: ""
+consumes:
+  events: []
+  rest_endpoints: []
+produces:
+  events: []
+  rest_endpoints:
+    - path: /secrets
+      method: POST
+    - path: /secrets/{secret_id}/envelopes/{client_pk}
+      method: GET
+storage:
+  vault: true
+config:
+  - server_private_key
+  - server_public_key
```

Comment:
What is this used for?

Reply:
The intend was to differentiate between inferrable and non-inferrable config variables. If we follow some standards when writing the services we can infer config parameters around kafka, fastapi, mongodb etc, but some services might require additional configuration that cannot be inferred from their formal description. Such configuration parameters can then be specified under `config`.

Reply:
OK I see. Should we maybe name it "extra_config" or something similar?

Reply:
👍